README.md

```diff
@@ -304,12 +304,22 @@ After preparing this directory, point to it when running the local proxy with th
 * Consider running the local proxy on separate hosts, containers, sandboxes, chroot jail, or a virtualized environment
 
 #### Access tokens
-
-* Access tokens are not meant to be re-used.
-* After localproxy uses an access token, it will no longer be valid.
+* After localproxy uses an access token, it will no longer be valid without an accompanying Client Token.
 * You can revoke an existing token and get a new valid token by calling [RotateTunnelAccessToken](https://docs.aws.amazon.com/iot/latest/apireference/API_iot-secure-tunneling_RotateTunnelAccessToken.html).
 * Refer to the [Developer Guide](https://docs.aws.amazon.com/iot/latest/developerguide/iot-secure-tunneling-troubleshooting.html) for troubleshooting connectivity issues that can be due to an invalid token.
 
+#### Client Tokens
+* The client token is an added security layer to protect the tunnel by ensuring that only the agent that generated the client token can use a particular access token to connect to a tunnel. 
+* Only one client token value may be present in the request. Supplying multiple values will cause the handshake to fail.
+* The client token is optional.
+* The client token must be unique across all the open tunnels per AWS account
+* It's recommended to use a UUIDv4 to generate the client token.
```

Comment:
Minor: I can imagine use cases for other types of uuid, not just v4. Perhaps drop `v4`.